### PR TITLE
Update CI configuration

### PR DESCRIPTION
**Pull request by itskingori**
Title says it all. See commits for details.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN apt-get update -y && \
     rm -rf /tmp/* /var/{cache,log,tmp} /var/lib/{apt,dpkg,cache,log}
 
 # Configure source path
-ARG SOURCE_PATH="/go/src/github.com/Intellection/port-control-cli"
+ARG SOURCE_PATH="/go/src/github.com/Intellection/sentry-events-fetcher"
```
Comment (zacblazic) [inline Dockerfile]: Whoops.

Comment (itskingori) [reply Dockerfile]: Haha, yeah.

File: Dockerfile [hunk 2]
```diff
@@ -26,5 +26,10 @@ RUN make dependencies
 RUN make build && \
     mv bin/sentry-events-fetcher /usr/local/bin/sentry-events-fetcher
 
+FROM debian:stretch-slim
+
+WORKDIR /
```
Comment (zacblazic) [inline Dockerfile]: Is this necessary?

Comment (itskingori) [reply Dockerfile]: It was before, but not now. Idea morphed quite a bit. 😅

Comment (itskingori) [reply Dockerfile]: Removed in https://github.com/Intellection/sentry-events-fetcher/pull/4/commits/929c48b819684815d653553e801609e976a9e204.

**Pull request merged**